src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,4 +1,76 @@
 databaseChangeLog:
-  -  changeSet:
-       id:  adicionar suporte ao liquibase
-       author:  izumizawa
\ No newline at end of file
+
+  - changeSet:
+      id: create_client_database_structure
+      author: Gabriel Oswaldo
+      changes:
+        - createTable:
+              tableName: marital_status
+              columns:
+                - column:
+                    name: marital_status_id
+                    type: int
+                    constraints:
+                      primaryKey: true
+                      primaryKeyName: marital_status_pk
+                - column:
+                    name: marital_status
+                    type: varchar(20)
+        - insert:
+            tableName: marital_status
+            columns:
+              - column:
+                  name: marital_status_id
+                  value: 1
+              - column:
+                  name: marital_status
+                  value: single
+        - insert:
+            tableName: marital_status
+            columns:
+              - column:
+                  name: marital_status_id
+                  value: 2
+              - column:
+                  name: marital_status
+                  value: married
+        - insert:
+            tableName: marital_status
+            columns:
+              - column:
+                  name: marital_status_id
+                  value: 3
+              - column:
+                  name: marital_status
+                  value: widower
+        - createTable:
+            tableName: client
+            columns:
+              - column:
+                  name: client_id
+                  type: UUID
+                  constraints:
+                    primaryKey: true
+                    primaryKeyName: client_pk
+              - column:
+                  name: marital_status_id
+                  type: int
+                  constraints:
+                    nullable: false
+                    foreignKeyName: client_marital_status_fk
+                    references: marital_status(marital_status_id)
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: rg
+                  type: varchar(15)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: cpf
```

Comment:
Maybe change to tax_id? (in english)

Reply:
good catch @gdseabra, these documents (RD, CPF) are for brazil use only, we can remove these two and create a "identity_document", associated with a "identity_document_type", we candiscuss it later :)

for now we can merge it like this and change it in a future task